File: VB/CustomPrintPreview/Grid/CustomGridControl.vb
Imports System
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator

Namespace CustomPrintPreview.Grid
    Public Class CustomGridControl
        Inherits GridControl

        Protected Overrides Sub RegisterAvailableViewsCore(ByVal collection As InfoCollection)
            MyBase.RegisterAvailableViewsCore(collection)
            collection.Add(New CustomGridViewInfoRegistrator())
        End Sub
    End Class
End Namespace

File: CS/CustomPrintPreview/Grid/CustomGridControl.cs
using System;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;

namespace CustomPrintPreview.Grid {
    public class CustomGridControl :GridControl {
        protected override void RegisterAvailableViewsCore(InfoCollection collection) {
            base.RegisterAvailableViewsCore(collection);
            collection.Add(new CustomGridViewInfoRegistrator());
        }
    }
}
